src/components/TimeFrameControl.tsx

```diff
@@ -13,7 +13,7 @@ export const Button: FC<{
   title?: string;
 }> = ({ children, isActive, onClick, title }) => (
   <button
-    className={`
+    className={` 
```

Comment:
Can you take out the commit with the space? 😄

Reply:
I reverted that commit